src/ducks/balance/Balance.jsx

```diff
@@ -136,6 +143,54 @@ class Balance extends PureComponent {
     })
   }
 
+  startRealtime(type, callback) {
+    if (!this.realtime[type]) {
+      const cozyClient = this.props.client
+      const url = cozyClient.client.uri
+      const token = cozyClient.client.token.token
+      this.realtime[type] = new CozyRealtime({ url, token })
+      this.realtime[type].subscribe({ type }, 'created', callback)
```

Comment:
I think we need only one CozyRealtime and not one CozyRealtime per doctype.

Ideally it should go like this

cozyClient.realtime.subscribe(doctype, token)

We should not care about cozyClient.client.uri  and .token.token inside the component, the initialisation of CozyRealtime from cozy-client should be extracted from the component as it will surely move and be refactored. The component does not care about that.